app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, old_file, path = command.split()
+    path_dirs = path.split("/")
+    new_file = path_dirs.pop(-1)
+
+    with open(old_file, "r") as file_out:
```

Comment:
Instead of opening one context manager inside of another, you could open two content managers at once.